spec/services/npq/void_participant_outcome_spec.rb

```diff
@@ -11,7 +11,7 @@
   let!(:participant_profile) do
     create(:npq_participant_profile, npq_lead_provider: cpd_lead_provider.npq_lead_provider, npq_course:)
   end
-  let(:participant_declaration) do
+  let!(:participant_declaration) do
```

Comment:
same with this spec this should go with your other pr as we're removing the whole endpoint/service